README.md

```diff
@@ -61,7 +61,7 @@ clusterrolebinding.rbac.authorization.k8s.io/glusterfs-csi-role created
 Below listed feature gates need to be enabled in kubernetes v1.12.1
 
 ```
---feature-gates=CSIPersistentVolume=true,MountPropagation=true,VolumeSnapshotDataSource=true,KubeletPluginsWatcher=true,CSINodeInfo=true,CSIDriverRegistry=true
+--feature-gates=VolumeSnapshotDataSource=true,"CSINodeInfo=true","CSIDriverRegistry=true"
```

Comment:
do we require double quotes? if yes we need to add it to `VolumeSnapshotDataSource=true` also

Reply:
yeah,  let me add.

Reply:
@Madhu-1 Done.

Reply:
@humblec, I don't think we need to add double quotes around feature gates
example from the kubernetes doc
`--feature-gates=BlockVolume=true,CSIBlockVolume=true`

Reply:
I think the effect is same, however I removed the quotes for parity between the docs. @Madhu-1 ptal.